bigquery/storage/managedwriter/integration_test.go

```diff
@@ -504,6 +509,144 @@ func testCommittedStream(ctx context.Context, t *testing.T, mwClient *Client, bq
 		withExactRowCount(int64(len(testSimpleData))))
 }
 
+// testSimpleCDC demonstrates basic Change Data Capture (CDC) functionality.   We add an initial set of
+// rows to a table, then use CDC to apply updates.
+func testSimpleCDC(ctx context.Context, t *testing.T, mwClient *Client, bqClient *bigquery.Client, dataset *bigquery.Dataset) {
+	testTable := dataset.Table(tableIDs.New())
+
+	if err := testTable.Create(ctx, &bigquery.TableMetadata{
+		Schema: testdata.ExampleEmployeeSchema,
+		Clustering: &bigquery.Clustering{
+			Fields: []string{"id"},
```

Comment:
Why is this needed for CDC ?

Reply:
CDC functional requires the table to have a primary key, and clustering setup for it to reduce scans.

Without it, you get an error of the flavor `The primary keys and clustering keys of the table project.dataset.table are required to create an upsert stream.`